repairnator/repairnator-dockerpool/src/main/java/fr/inria/spirals/repairnator/dockerpool/Launcher.java

```diff
@@ -236,15 +238,20 @@ private void printUsage() {
         System.exit(-1);
     }
 
-    private List<Integer> readListOfBuildIds() {
-        List<Integer> result = new ArrayList<>();
+    private Map<Integer, Integer> readListOfBuildIds() {
```

Comment:
Sorry but it's kind of ugly. We should have a data structure defined, for example, in repairnator-core that specify an InputId which is a simple single int when using repair mode and a couple of id when using bears mode.

Reply:
Don't be sorry, I think that ugly too, I just didn't think in another way. I was more concerned at doing the docker image to work. I will try your approach and then we see how it looks like.

Reply:
Forget my last message.